setup.py

```diff
@@ -13,6 +13,7 @@
     test_suite='test_quickcache',
     install_requires=[
         'six==1.11.0',
+        'pytz==2018.3',
```

Comment:
This may seem silly, but I'd like to keep quickcache's footprint as small as possible. Since the dependency on `pytz` is really just to get access to a UTC timezone, I suggest just inlining the top of the first example here: https://docs.python.org/2.7/library/datetime.html#tzinfo-objects into a file called `native_utc.py` or something, so it would just contain

```
from datetime import tzinfo, timedelta, datetime

ZERO = timedelta(0)
HOUR = timedelta(hours=1)

# A UTC class.

class UTC(tzinfo):
    """UTC"""

    def utcoffset(self, dt):
        return ZERO

    def tzname(self, dt):
        return "UTC"

    def dst(self, dt):
        return ZERO

utc = UTC()
```

Then, when you need `utc`, in the code you just added, you can import it from that file. This will make it possible for people to use `quickcache` without also using `pytz` or (maybe even more importantly) when pinned to a version of `pytz` other than `2018.3`. It may be a small gain in practice, but it's also a fixed small cost, and I think it's also good practice.

Reply:
ended up just dropping it completely